demos/s3server/s3server.py

```diff
@@ -42,12 +42,17 @@
 from tornado import ioloop
 from tornado import web
 from tornado.util import unicode_type
+from tornado.options import options, define
 
 try:
     long
 except NameError:
     long = int
 
+define("port", default=9888, help="TCP port to listen on")
+define("root_directory", default="/tmp/s3", help="Root storage directory")
+define("bucket_depth", default=0, help="Bucket file system depth limit")
+
 
 def start(port, root_directory="/tmp/s3", bucket_depth=0):
```

Comment:
Since you're using `define` for these parameters, it would make more sense to not give defaults that are never used here.

Reply:
Sorry, my English is poor, and I can't understand what you mean! 
Can I remove the default value in the start() function?

Reply:
Precisely. At the bottom of the file, you are explicitly passing all the parameters, so the default values as defined in `start` are extraneous.